README.md

```diff
@@ -1836,6 +1836,9 @@ IEEE Trans. Multimedia (TMM), 2020.
 arXiv, 2020.
 - [Baldwin et al., arXiv 2021](#Baldwin21arxiv).  
 *Time-Ordered Recent Event (TORE) Volumes for Event Cameras*.
+- Alkendi Y., Azzam R., Ayyad A., Javed S., Seneviratne L., and Zweiri Y.
+[Neuromorphic Camera Denoising using Graph Neural Network-driven Transformers](https://arxiv.org/abs/2112.09685). 
+arXiv, 2021. [PDF](https://arxiv.org/abs/2112.09685). [Youtube](https://www.youtube.com/watch?v=x0FXZLEenJ8), [Dataset](https://github.com/Yusra-alkendi/ED-KoGTL)
```

Comment:
thanks. Now you need to add two blank spaces at the end of the line to force a line break on github.
see the current formatting issue: 
![image](https://user-images.githubusercontent.com/8024432/147580283-614e2a6a-b098-4fcf-810a-7108c7c66937.png)
Please see the preview before committing

Reply:
Hi, 
Thank you. 
I have updated the Readme.md.

Reply:
Hi. Are you sure? I still see the same thing:
![image](https://user-images.githubusercontent.com/8024432/147582115-37336df9-4dd3-4bc5-84f3-accba119ff42.png)